Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift

```diff
@@ -26,9 +26,14 @@ struct SurveyDetailView: View {
 
     var body: some View {
         ScrollView(.vertical, showsIndicators: false) {
-            ZStack(alignment: .topLeading) {
-                setUpMainImage()
-                setUpComponents()
+            LoadingView(
```

Comment:
I didn't learn this technique yet when I review this view, but another way to define LoadingView is something [like this](https://github.com/blyscuit/kmm-swiftUI-IC/blob/c387b8d6164fdd28983ea8b033cd828b711ba808/iosApp/Survey/Sources/Presentation/ViewModifiers/LoadingDialog.swift). Only want to share to you so that you have more viewpoints. 😄

Reply:
Cool cool approach 👍 , I think working with `ViewModifier` is more "SwiftUI" than mine 😄.
Really thanks for sharing the new way :D.